source/fundamentals/aggregation/aggregation-expression-operations.txt

```diff
@@ -40,45 +42,71 @@ guide, consider importing the following classes in your program:
 method, such as ``getString()`` or ``getDate()``. The following code
-shows how to reference an integer field called ``quantity``:
+shows how to reference a string field called ``name``:
 
 .. code-block:: java
    :copyable: true
+   
+   current().getString("name")
 
-   var myField = current().getInteger("quantity");
-
-To reference a constant value, use the ``of()`` method, which converts
-primitive types to types that can be used in aggregation operations. The
-following code shows how to reference a constant of value ``1.0``:
+To specify a value in an operation, pass it to the ``of()`` method to
+convert it to a valid type. The following code shows how to reference a
+value of ``1.0``:
 
 .. code-block:: java
    :copyable: true
+   
+   of(1.0)
 
-   var myNum = of(1.0);
+To create an operation, chain a method to your field or value reference.
+You can build more complex operations by chaining additional methods.
 
-When you finish composing an aggregation operation, you can pass it to
-an aggregation builder method, such as ``match()`` or ``project()`` to
-complete an aggregation pipeline stage. To learn more about these methods,
-see :ref:`Aggregates Builders <aggregates-builders>`.
+The following example creates an operation to find patients in New
+Mexico who have visited the doctor’s office at least once. The operation
+performs the following actions:
 
-.. note::
-   
-   While these new operations can be used within existing aggregation
-   methods that take ``TExpression`` values, methods that return
-   ``Bson`` values are not usable with this feature.
+- Checks if the size of the ``visitDates`` array is greater than ``0``
+  by using the ``gt()`` method 
+- Checks if the ``state`` field value is “New Mexico” by using the
+  ``eq()`` method
+
+The ``and()`` method links these operations so that the pipeline stage
+matches only documents that meet both criteria.
+
+.. code-block:: java
+   :copyable: true
+      
+   current()
+         .getArray("visitDates")
+         .size()
+         .gt(of(0))
+         .and(current()
+                 .getString("state")
+                 .eq(of("New Mexico")));
+
+In most cases, you need to include your operation in a method such as
+``computed()`` or ``expr()`` to process it for use in an aggregation
+pipeline stage.
+
+To create a usable aggregation pipeline stage, include your expression
+in an aggregates builder method such as ``match(<expression>)`` or
+``project(<expression>)``. To learn more about these methods, see
+:ref:`Aggregates Builders <aggregates-builders>`.
+
+The examples use the ``Arrays.asList()`` method to create a list of
+aggregation stages that is passed to the ``aggregate()`` method of
+``MongoCollection``.
+
+Methods such as ``computed()`` and ``expr()`` that take values of type
+``TExpression`` allow you to use your expressions in aggregations. For
+example, the ``expr()`` method processes your operation as an expression
+that the ``match()`` method will accept.
```

Comment:
I wonder if this should be in a note at line 96 instead.

Reply:
Considering my earlier comment, I would be inclined to agree. Putting this down here creates a disconnect from the knowledge. I don't know if it needs to be a tip, but I would definitely move it up.

Reply:
made a note, LMK if you think the flow of info is good this way or if the content should be included as regular text.

Reply:
I think this works just fine as a note.